go.mod

```diff
@@ -2,132 +2,148 @@ module github.com/oasisprotocol/nexus
 
 go 1.19
```

Comment:
Hm i thought oasis-core/go v 23 requires go 1.21, apparently not? Probably good idea to bump in a followup PR.

Reply:
:shrug:  apparently.

Upgrade: Not opposed to it but I don't see a benefit to the busywork unless it's needed?